snippets/multisign.py

```diff
@@ -57,11 +58,10 @@
 if multisigned_tx_response.result["validated"]:
     print("The multisigned transaction was accepted by the ledger:")
     print(multisigned_tx_response)
-    if multisigned_tx_response.result["Signers"]:
-        print(
-            "The transaction had "
-            f"{len(multisigned_tx_response.result['Signers'])} signatures"
-        )
+    signers_in_response = multisigned_tx_response.result["tx_json"]["Signers"]
```

Comment:
```suggestion
    signers_in_response = multisigned_tx_response.result["tx_json"].get("Signers")
```
Otherwise, the code will error instead of hitting the if.